pkgs/applications/virtualization/cri-o/default.nix

```diff
@@ -28,8 +28,7 @@ buildGoPackage rec {
     rev = "v${version}";
     sha256 = "142flmv54pj48rjqkd26fbxrcbx2cv6pdmrc33jgyvn6r99zliah";
   };
-
-  outputs = [ "out" "man" ];
```

Comment:
We probably want to keep this are we are installing manpages.

Reply:
Alright, readded it.